scripts/pre_commit_linter.py

```diff
@@ -739,8 +739,14 @@ def _pre_commit_linter(all_files):
     for file_group, process in zip(file_groups_to_lint, linting_processes):
         # Require timeout parameter to prevent against endless waiting for the
```

Comment:
Probably move this comment inside the try block?

Reply:
Done, thanks!